<p>
Several days ago, Yuri Deigin published a very detailed and extremely well-researched article</a>, which makes a compellingly strong argument that the Coronavirus
(SARS-CoV-2) may have indeed been engineered. Here is a (not so concise) summary.
</p>

<p>
<a href="https://medium.com/@yurideigin/lab-made-cov2-genealogy-through-the-lens-of-gain-of-function-research-f96dd7413748">Yuri's article</a> is fairly long and 
complex. It's a much deeper scientific review of the claim that was previously discussed <a href="https://fitfortis.com/newsfeed/1631285e-0a02-457f-d0c9-08d7ebd18c05?lang=en-us">
here</a>. Below is a digest from the article, and from other sources.
</p>

<h1>The basics</h1>
<p>
The genome of SARS-CoV-2 is well understood and discussed
<a href="https://fitfortis.com/newsfeed/83e785af-8b79-48b0-952f-08d7e0f0ffb0?lang=en-us">here</a>:
</p>

<p>
<ul>
<li>The viral RNA contains about <b>30 kilobases</b>, encoding <b>29 proteins</b>. </li>
<li>Of them, 4 are <b>structural proteins</b> (S - spike, E - envelope, M - membrane,
and N - nucleocapsid). The other 25 are <b>utility proteins</b>.</li>
<li>The S protein enables the virion to attach to the victim cell, by
using its ACE2 receptor.</li>
<li>The gene of the S protein comprises about 12-13% of the viral genome, and  consists of 2 parts: 
<b>S1 and S2</b>. S1 interacts with the ACE2 receptor. S2 is responsible for fusion with the cell membrane and 
penetration into the cell.</li>
<li>The part of S1 that interacts with ACE2 is called <b>Receptor
Binding Domain (RBD)</b>. The area of direct contact is called <b>Receptor Binding Motif
(RBM)</b>.</li>
</ul>
</p>

<p>
The RBM is a key part of the puzzle. It looks as follows:
</p>
<p>
<img 
  src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.19.956235/F1.large.jpg?width=800&height=600&carousel=1"
  width="800"
  >
</img>
</p>
<p>
<cite>Overall structure of the SARS-CoV-2 S protein, bound with the ACE2 receptor of the victim cell: 
a) shows position of RBM within the S gene, b) shows the sequence of the RBD, c) shows the structure of 
the RBD bound with ACE2. The RBD core is colored cyan. 
</cite>
</p>

<h1>Cast of characters</h1>
<p>There are humans in this story...</p>
<p>
<ul>
  <li><b><a href="https://sph.unc.edu/adv_profile/ralph-s-baric-phd/">Prof. Dr. Ralph Baric</a></b> -- A legend in coronavirology, introducing a number of synthetic genomic manipulation techniques. 
    Worked closely with Shi Zheng-Li on gain-of-function (GOF) research.</li>
  <li><b><a href="https://en.wikipedia.org/wiki/Shi_Zhengli">Dr. Shi Zheng-Li</a></b> -- Dr Shi Zheng-Li is the leader of coronavirus research at the Wuhan Institute of Virology. Also known as 
    "bat woman" for her research on coronaviruses in bats, she has worked under the supervision of Peter 
    Rottier, and in collaboration with Ralph Baric.</li>
  <li><b><a href="https://www.researchgate.net/profile/Peter_Rottier">Prof. Dr. Peter Rottier</a></b> -- Professor at Utrecht University, who supervised the work of Dr Shi Zheng-Li on coronaviruses in the early 2000s.</li>
</ul>
</p>

<p>... and then, there are viruses...</p>
<p>
<ul>
  <li><b>SARS-CoV-2</b> -- The new coronavirus, causing COVID-19.</li>
  <li><b>RaTG13</b> -- A coronavirus found in bats, which is the obvious ancestor of SARS-CoV-2. Its genome 
    is 96% equivalent to that of SARS-CoV-2.
  </li>
  <li><b>MP789</b> -- A coronavirus found in 29 pangolins, which were confiscated
    by customs from smugglers in the Guangdong province. All 29 pangolins
    experienced death in captivity, due to the virus, despite the
    efforts of the rescue workers.</li>
</ul>
</p>


<h1>Sequence of events</h1>
<p>
First let's look at the sequence of events:
</p>
<p>
<table>
<thead>
<tr>
  <th>When</th>
  <th>What happened</th>
</tr>
</thead>
<tbody>

<tr>
<td>February,&nbsp;2000</td>
<td>The Dutch group of Peter Rottier from Utrecht University publishes a 
  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC111474/">paper</a> demonstrating 
  creation of a coronavirus that can cross host cell species barrier, by substitution of parts 
  of its S protein.
</td>
</tr>

<tr>
  <td>2002</td>
  <td>
    Ralph Baric and his group publish a <a href="https://jvi.asm.org/content/76/21/11065">breakthrough 
    paper</a>, demonstrating the first synthetic assembly of a natural murune coronavirus.
  </td>
</tr>

<tr>
  <td>November&nbsp;16,&nbsp;2002</td>
  <td>SARS-CoV epidemic outbreak identified in Foshan, Guangdong, China.</td>
</tr>

<tr>
  <td>October&nbsp;28,&nbsp;2003</td>
  <td>Ralph Baric and his group publish a <a href="https://www.pnas.org/content/100/22/12995">paper</a> 
  on synthetically recreating SARS-CoV. What is notable here is how quickly a qualified team of virologists 
  can re-create a virus synthetically.</td>
</tr>

<tr>
<td>2005</td>
<td>Shi Zheng-Li, Peter Rottier and others publish a <a href="https://jvi.asm.org/content/79/22/14451">paper</a> 
discussing what allows viruses to expand their host range.</td>
</tr>

<tr>
  <td>January 2007</td>
  <td>Ralph Baric and his group publish a <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1769406/">paper</a> 
  describing a lab-modified human SARS-CoV, improved in mice, which turned to be a highly virulent and lethal virus.</td>
</tr>

<tr>
<td>2007</td>
<td>Shi Zheng-Li's group at the Wuhan Institute of Virology (WIV) publishes a study 
  of the S protein of human and bat coronaviruses. 
</td>
</tr>

<tr>
  <td>2013</td>
  <td>Shi Zheng-Li's team isolates RaTG13 from bats in a Yunnan cave. However, outside of 
  her lab, no one knew about that strain until January 2020.</td>
</tr>
<tr>
  <td>October&nbsp;30,&nbsp;2013</td>
  <td>Shi Zheng-Li's group publishes a <a href="https://www.nature.com/articles/nature12711">paper in Nature</a> 
  announcing that in Yunnan caves they had discovered carrier bats of the RsSHC014 and Rs3367 strains 
  that coincided with the first SARS-CoV by 85% and 96%, respectively. Shi Zheng-Li gets 
  the moniker "bat woman".
  </td>
</tr>

<tr>
  <td>2015</td>
  <td>Shi Zheng-Li and Ralph Baric publish a <a href="https://www.nature.com/articles/nm.3985">joint paper in Nature</a>, 
    which becomes the most famous gain-of-function (GOF) virology paper.</td>
</tr>

<tr>
  <td>February&nbsp;18,&nbsp;2016</td>
  <td>Shi Zheng-Li publishes a <a href="https://www.ncbi.nlm.nih.gov/pubmed/26920708https://www.ncbi.nlm.nih.gov/pubmed/26920708">paper</a> on RaBtCoV/4991 - a new coronavirus that's practically
  identical to MP789.
  </td>
</tr>

<tr>
  <td>March&nbsp;15,&nbsp;2016</td>
  <td>The group of Ralph Baric in collaboration with Shi Zheng-Li publishes a <a href="https://www.pnas.org/content/113/11/3048.full">paper</a> 
  on insertion of a S protein segment into a mouse-adapted SARS.
  </td>
</tr>

<tr>
  <td>November&nbsp;30,&nbsp;2017</td>
  <td>Shi Zheng-Li's group publishes a <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5708621/">paper</a> 
    on the creation of 8 new viruses, transplanting RBDs from bat SARS-like viruses, collected from the cave in 
    Yunnan. </td>
</tr>

<tr>
  <td>2017</td>
  <td>Another batch of pangolins get confiscated, with a similar virus (Pangolin P2V/2017).
  </td>
</tr>
<tr>
<td>March-December 2019</td>
<td>
4 Chinese pangolins (Manis pentadactyla) and 25 Malayan <a href="https://miro.medium.com/max/494/1*VrsIOqvp6ee8IfyU4LSimg.png">pangolins</a> (Manis javanica)
are confiscated by the Customs and Department of Forestry of Guangdong Province from
smugglers. These animals were sent to the wildlife rescue center in Guangdong, and were mostly inactive and
sobbing, and eventually died in custody of severe coronavirus symptoms, despite exhausting
rescue efforts. Tissue samples were taken from the lung, lymph nodes, liver, spleen, muscle,
kidney, and other tissues from pangolins that had just died for histopathological and
virological examinations. The coronavirus from the pangolins (MP789) gets isolated.
</td>
</tr>

<tr>
<td>October 2019</td>
<td>
Chinese scientists in Beijing <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6832359/"> demonstrate</a>
insertion of furin into a live chicken coronavirus - infectious bronchitis virus. As a side note 
the addition of a furin site allows the mutant virus to infect nerve cells (loss of smell, 
anyone?)
</td>
</tr>

<tr>
  <td>October 2019</td>
  <td>The group of Ralph Baric submits a <a href="https://jvi.asm.org/content/94/5/e01774-19">paper</a> on 
  the importance of S protein protease cleavage (see Furin below) to crossing the barrier to zoonotic infection 
  by coronaviruses.</td>
</tr>


<tr>
  <td>January&nbsp;10,&nbsp;2020</td>
  <td>The SARS-CoV-2 genome is sequenced and made <a href="https://www.ecdc.europa.eu/en/novel-coronavirus/event-background-2019">publicly available</a>.</td>
</tr>
<tr>
  <td>January&nbsp;23,&nbsp;2020</td>
  <td>Shi Zheng-Li releases a <a href="https://www.biorxiv.org/content/10.1101/2020.01.22.914952v2.full">paper</a> indicating that CoV2 is 96% identical to RaTG13, a strain which her laboratory had previously isolated from Yunnan bats in 2013. 
  The paper gets published in <a href="https://www.nature.com/articles/s41586-020-2012-7">Nature</a> too.
  </td>
</tr>
<tr>
<td>February&nbsp;13,&nbsp;2020</td>
<td>
A team in Hong Kong releases a <a href="https://www.biorxiv.org/content/10.1101/2020.02.13.945485v1.full">paper</a> that notes
clear signs of recombination in the S protein of SARS-CoV-2.
</td>
</tr>
<tr>
<td>February&nbsp;17,&nbsp;2020</td>
<td>
Chinese scientists <a href="https://www.biorxiv.org/content/10.1101/2020.02.17.951335v1">discover</a>
a peculiar strain of pangolin coronavirus in their possession, which, while generally being only 90%
similar to CoV2, in the RBM region was almost identical to it, with only a single amino acid
difference.
</td>
</tr>
<tr>
  <td>February&nbsp;21,&nbsp;2020</td>
  <td>
    Swiss scientists publish a <a href="https://www.biorxiv.org/content/10.1101/2020.02.21.959817v1">paper</a>, 
    demonstrating creation of a synthetic clone of SARS-CoV-2. 
  </td>
</tr>

</tbody>
</table>
</p>

<h1>The smoking gun...</h1>
<p>
Now, let's look at genome similarities. This first chart shows that
the genome of the bat coronavirus RaTG13 (in blue) has a very high level of identity to the
genome of SARS-CoV-2. The original 2003 SARS-CoV virus (in red) is quite
different genetically.
</p>

<p>
<img 
  src="https://miro.medium.com/max/1400/1*Zdb3coRedsTCBkrvp7bQjw.png"
  width="800">
</img>
</p>
<p>
<cite>Levels of genome similarity between the new Coronavirus (SARS-CoV-2), the 
original SARS virus from 2003 (SARS-CoV, in red), and the bat coronavirus (RaTG13, in blue).
</cite>
</p>

<p>
There's a notable dip in similarity around the S protein encoding, and
more specifically in the area of the RBM. At the same time, the pangolin
coronavirus strain (2019-nCoV-WIV02 / MP789) has a RBM that almost perfectly matches
what we see in SARS-CoV-2:
</p>

<p>
<img 
  src="https://miro.medium.com/max/1400/1*k7X4kwOODUo6nMM6QIhypA.png"
  width="800"
  >
</img>
</p>

<p>
The red line above is the line of genome similarity of the pangolin
coronavirus (2019-nCoV-WIV02 / MP789) and SARS-CoV-2.
</p>

<p>
In other words, there must have been some strange interaction between bats
(from Yunnan) and pangolins (from Guangdong), with exchange of bodily fluids,
even though they are naturally separated by more than
<a href="https://goo.gl/maps/XGpVZQxFG8RJ7pKe8">1100 miles</a> (!!!).
</p>

<h1>Furin</h1>
<p>Last but not least - the <b>furin cleavage site</b>. Here is the detailed
diagrams of the S protein gene. One of the key additions in the SARS-CoV-2 genome
is the furin site between S1 and S2. <a href="https://en.wikipedia.org/wiki/Furin">Furin</a>
is the most effective protease -- an enzyme that cuts RNA, thus <b>increasing
greatly the virality of the virus</b>, as demonstrated in
<a href="https://www.tandfonline.com/doi/full/10.1080/22221751.2020.1756700">this study</a>.
</p>
<p>
<img 
  src="https://miro.medium.com/max/1400/1*UjgUgWsM5dc3Yo-6Rj21Zw.png"
  width="800"
  >
</img>
<p>
<cite>
Topological diagram of the S protein gene.
</cite>
</p>

</p>
The encoding for furin is just 12 bases long (<code>ccucggcgggca</code>). It has been 
<a href="https://www.tandfonline.com/doi/full/10.1080/22221751.2020.1756700">recently 
demonstrated</a> that the absence of furin greatly decreases the pathogenicity and 
replication ability of the virus. 
<p>

<p>Here's a visual representation of the phylogenetic tree of coronavirus genomes. 
It is interesting that SARS-CoV-2 is the only virus from its closest family, with furin.
The closest relative with furin is BAT-CoV-HKU5, isolated in 2014 by Shi Zheng-Li's team. 
It's a very distant relative though -- the S proteins of SARS-CoV-2 and BAT-CoV-HKU5 
share only 36%. 
</p>
<p>
<img 
  src="https://miro.medium.com/max/1400/1*D0rOGYUmAzVnyM4KhbKoHw.png"
  width="800"
  >
</img>
</p>
<p>
<cite>
Phylogenetic tree of coronaviruses. SARS-CoV-2 is shown in the top left part of the the 
diagram, as the only virus of its family possessing furin. Viruses possessing furin are 
marked in pink.
</cite>
</p>

<h1>How hard is recombination?</h1>
<p>
Genetic recombination is the exchange of genetic material. It occurs naturally. It can also 
be done artificially by using enzymes. These days, artificial recombination is fairly easy. There 
are many companies that are selling recombination kits, like <a href="https://www.thermofisher.com/order/catalog/product/A13288#/A13288">this 
one</a>.   
</p>

<h1>How fast is mutation?</h1>
<p>
Natural (in vivo) mutations that would lead to the 4% difference in the genomes of SARS-CoV-2 and RaTG13 would 
take years. However, lab (in vitro) mutations can be performed much faster, as the passage can be sped up. A 
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC506962/">paper</a> from 2004 reported achieving a 2.1% 
difference in genomic sequences of spike protein, after only 600 passages. 
</p>


<h1>Putting it all together</h1>
<p>
So we have a situation where the new Coronavirus (SARS-CoV-2) is very similar (96%) to another virus 
(RaTG13) isolated from bats in 2013 (but unknown to the world until January 2020) <b>EXCEPT</b> in the RBM region 
of its spike protein. The RBM region of SARS-CoV-2 almost perfectly matches (with the exception of a single 
amino acid) what we see in a third virus (2019-nCoV-WIV02 / MP789), which otherwise is quite different from SARS-CoV-2.  
</p>

<p>
There's a notable insertion of furin - which makes the virus a lot more viral. 
</p>

<p>
And, to top it all off, the 2 ancestor viruses (the "bat virus" and the "pangolin virus") were found in 
2 very different species (bats and pangolins) more than 1100 miles away from each other. Note that bats are 
not sold in the Wuhan wet market where the outbreak originated.
</p>

<p>
All of this clearly indicates the strong possibility of <b>human-driven recombination</b> -- the
act of combining different parts of different RNAs in the same RNA strand.  Combine that with a steady 
history of lab leaks (H1N1 in 1977; SARS-CoV in summer of 2003, in December 2003, and spring of 2004; etc. - 
lab leaks are unfortunately quite common) and you might just get yourself a nice global pandemic... 
</p>

<p align="center">***</p>

<p>
  <cite>
    Sources: Yuri Deigin ("Lab-Made? SARS-CoV-2 Genealogy Through the Lens of Gain-of-Function Research"), Wikipedia, Fit Fortis
  </cite>
</p>